samples/inputs/date-time-input/input-format-display-format/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "react-date-time-input-format",
+  "description": "This project provides example of Date Time Input Format using Infragistics React components",
+  "author": "Infragistics",
+  "homepage": ".",
+  "version": "1.4.0",
+  "private": true,
+  "scripts": {
+    "start": "set PORT=4200 && react-scripts --max_old_space_size=10240 start",
+    "build": "react-scripts --max_old_space_size=10240 build ",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "dependencies": {
+    "igniteui-dockmanager": "1.14.2",
+    "igniteui-react": "18.6.0",
+    "igniteui-react-core": "18.6.0",
```

Comment:
Perhaps these packages should be updated to be aligned with the other samples in the repo:
```json
"igniteui-dockmanager": "1.14.3",
"igniteui-react": "18.6.1-alpha.0",
"igniteui-react-core": "18.6.1-alpha.0"
```
This applies for all the samples.